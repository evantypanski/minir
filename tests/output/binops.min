func main() -> none {
  debug(1 + 2);
  debug(1 + 2 * 3);
  debug(5 - 1);
  debug(5 / 2);
  let i = 40;
  debug(i + 2);
  ret;
}
