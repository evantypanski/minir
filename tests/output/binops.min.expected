3
7
4
2
42
